docker/Dockerfile.oldev

```diff
@@ -0,0 +1,59 @@
+FROM olbase 
+
+# Expose Open Library, Infogami, and SOLR
+EXPOSE 80 7000 8983
```

Comment:
I believe `EXPOSE` is inherited, so this can be removed.

Reply:
I wanted to keep it there to document which ports that particular container expected, so each container could technically stand alone.